Added displays for death count and last IL time

Thermospore opened the pull request.

Thermospore committed:
Reformat memory addresses

Lined up addresses for increased readability. Sorted ascending by address.

Thermospore committed:
Capability to split for escape

Splits between Dr. Fetus and Escape when IL splits are enabled

Thermospore committed:
Add death count display

Thermospore committed:
Added Last IL Time display

Thermospore committed:
Comment headers in split section

Increased readability of the split section by adding headers for each type of split

Thermospore committed:
Improved formatting of setting declarations

negative-seven reviewed (dismissed):
[removed]

negative-seven commented on LiveSplit.SuperMeatBoy.asl:
```diff
@@ -87,11 +92,61 @@ init
+		s = t;
+		
+		// Display final string
+		vars.SetTextComponent(name, s, settings["createUI"]);
```
You can use `String.Format("{0:0.000}", f)`

Thermospore replied:
Thanks! Yea, I suspected there was a more efficient way to do that...

Thermospore replied:
Fixed!

negative-seven commented on LiveSplit.SuperMeatBoy.asl:
```diff
@@ -29,6 +30,9 @@ startup
 	}
+		
+	settings.Add("deathDisp", false, "Death count display");
+	settings.Add("createUI", true, "Create UI if needed");
```
Could there perhaps be a way to omit needing createUI? Seems not too user-friendly the way it is now.

Thermospore replied:
Yea this setting is a holdover from a previous implementation, I suppose it isn't neccessary. I can just remove it

Thermospore replied:
Fixed!

negative-seven commented on LiveSplit.SuperMeatBoy.asl:
```diff
@@ -23,20 +23,16 @@ startup
 	settings.Add("ilDisp", false, "Last IL Time display");
-	settings.SetToolTip("ilDisp", "Times are truncated to 3 places (The game shows times rounded to two)");
-	
+		settings.SetToolTip("ilDisp", "Times are truncated to 3 places (The game shows times rounded to two)");
```
I'm a bit confused by these formatting changes?

Thermospore replied:
I arranged the settings in a hierarchy, so any settings/code that are a subset of their parent setting are indented

Thermospore replied:
I put the address list in ascending order with the addresses all lined up, which is a format I've found to be very readable and easy to work with

negative-seven replied:
The address list is a nice change, but additional indenting I think can be a bit misleading, making it look like a code block. I'll merge this right now, though.

Thermospore committed:
Increased efficiency of Last IL Time formatting

credit to -7 for this suggestion

Thermospore committed:
Removed createUI option

Thermospore committed:
Fixed bug with Last IL Time display

The Livesplit text component for the Last IL Time display would initialize on game launch even when the `ilDisp` setting was disabled (though it would only update the display if the setting was enabled)

negative-seven commented:
Thanks for contributing!
